distribution/docs/src/main/jdocs/content/_configuring/configuring-global-settings-contents.adoc

```diff
@@ -372,7 +372,8 @@ nfs.conf,
 ftpd.conf,
 ntp.conf,
 web.config,
-robots.txt
+robots.txt,
+Thumbs.db
```

Comment:
It's not dangerous, but on Windows machines I believe this gets auto generated and will continuously be created and ingested through the CDM